scss/components/_utils.scss

```diff
@@ -2,3 +2,11 @@
   position: relative;
   top: -1px;
 }
+
+.civicase__text-warning {
```

Comment:
Warning and success is already the part of the bootstrap theme. 
`text-warning` and `text-success` are the classes FYI.

Reply:
Yes but the colors different than design. Shoreditch is not using `brand-success` and `brand-warning` variables for these two classes. forgot to mention this in the PR, apologies.

Reply:
Oh. Yes. Np. LGTM